certified-connectors/LAWLIFT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,660 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LAWLIFT",
```

Comment:
@makaveli1313 It is not allowed to keep the connector's title in capital letters, Can you please use camel-case for your connector's name anywhere in the connector artifacts